gpio/__init__.py

```diff
@@ -79,23 +74,30 @@ def parse_item(self, item):
             GPIO.setup(in_pin, GPIO.IN, pull_up_down=GPIO.PUD_UP)
             self.logger.debug("GPIO: INPUT {0} assigned to pin \'{1}\'".format(item, in_pin))
             self._items.append(item)
+            self._itemsdict[in_pin] = item
             return self.update_item      
         if self.has_iattr(item.conf, 'gpio_out'):
             out_pin = int(self.get_iattr_value(item.conf, 'gpio_out'))
+            GPIO.setup(out_pin, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+            self._itemsdict[out_pin] = item
+            value = GPIO.input(out_pin)
+            item(value, 'GPIO', 'gpio_init')
+            GPIO.add_event_detect(out_pin, GPIO.BOTH, callback=self.get_sensors)
```

Comment:
As far as I see it, it works for out pin but I am not totally sure. At least it is possible to query the current value of an output pin with GPIO.input